internal/lookout/repository/jobs.go

```diff
@@ -208,6 +214,7 @@ func rowsToJobs(rows []*JobRow) ([]*lookout.JobInfo, error) {
 					Job:       job,
 					Cancelled: ParseNullTime(row.Cancelled),
 					JobState:  state,
+					Reason:    reason,
```

Comment:
I think that the `reason` should be attributed to job runs instead of jobs. E.g. a job could have multiple runs (executions), but only one of them might be pre-empted.

Imo the column should be added to the job_run table, and on the UI it should be displayed as part of the Job run information, only if not null.

Reply:
I thought about that @carlocamurri but it seemed to me this was compatible with "JobState" as a job column -- state is also derived from the job_run info but it is convenient here as a cache of "most recent state".

Reply:
I agree with Carlo. It sounds more natural to have it be part of job runs. Since it's the run that gets preempted. The job as a whole is only failed if we decide to not try running the job after preemption.

Reply:
This is only showing reason for job failed event which is the final run, yes?

Reply:
I left reason here because it now comes from JobErrors pulsar event - there is no associated job run on that event type, I guess because there are errors that may precede scheduling. Preempted is now a timestamp field on job_run, corresponding to that event.

Reply:
Removed "reason" column entirely, using only job_run.preempted timestamp now